core/templates/pages/topic-viewer-page/story-node-tile/story-node-tile.component.html

```diff
@@ -0,0 +1,213 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<mat-card class="story-node-card" *ngIf="nodeDataLoaded">
+  <div class="story-node-image">
+    <img [src]="pathIconParameters[0].thumbnailIconUrl"
+         [style.backgroundColor]="pathIconParameters[0].thumbnailBgColor"
+         alt="Icon"
+         class="image">
+  </div>
+  <div class="story-node-contents">
+    <h3 class="node-title">Lesson: {{node.getTitle()}}</h3>
+    <p class="node-description">{{ node.getDescription() }}</p>
+    <div class="progress-bar-container" [ngClass]="{'single-checkpoint-bar': checkpointCount === 1, 'multiple-checkpoints-bar': checkpointCount > 1}">
+      <div class="progress-bar">
+        <div class="completed-progress-bar" [style.width.%]="getCompletedProgressBarWidth()"></div>
+      </div>
+      <div *ngFor="let checkpointNode of checkpointStatusArray; index as idx"
+                [ngClass]="{'completed-checkpoint-node': checkpointStatusArray[idx] === 'completed', 'in-progress-checkpoint-node': checkpointStatusArray[idx] === 'in-progress', 'incomplete-checkpoint-node': checkpointStatusArray[idx] === 'incomplete'}">
+      </div>
+    </div>
+    <button class="node-button" *ngIf="resumeButtonActivate">resume</button>
+    <button class="node-button" *ngIf="!resumeButtonActivate" (click)="clickstart()">start</button>
```

Comment:
Please rename the method, it should describe the action a bit. Somewhere around ```StartLesson()```?